pentaho/impl/pom.xml

```diff
@@ -90,11 +90,6 @@
       <groupId>net.sf.ehcache</groupId>
       <artifactId>ehcache-core</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Running `mvn clean test` in **pentaho/impl**:
- [INFO] Tests run: 38, Failures: 0, Errors: 0, Skipped: 0

All tests run without jackson. @graimundo